west.yml

```diff
@@ -86,7 +86,7 @@ manifest:
       revision: 4bff01084d225996e4aae84b98be5969e2f9f33d
       path: tools/net-tools
     - name: hal_nxp
-      revision: 3e3f20e7de7606d735cdeefe21d5f2a51bf06095
+      revision: pull/34/head
```

Comment:
Please update to the new SHA now that zephyrproject-rtos/hal_nxp#34 has been merged

Reply:
I've done it, rebased the branch on master and fixed the conflicts. It is ready to be merged.